demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -284,6 +285,27 @@ def _label_as_optional(pack_ids: set) -> list:
         """
         return [(p, False) for p in pack_ids]
 
+    @staticmethod
+    def _update_optional_commontypes_pack_dependencies(packs_found_from: set) -> list:
+        """
+        Updates pack_dependencies_data for optional dependencies, excluding the CommonTypes pack.
+
+        Args:
+            packs_found_from (set): pack names.
+
+        Returns:
+            packs_found_from (list): representing the dependencies.
+
+        """
+        common_types_pack_dependency = False
+        if COMMON_TYPES_PACK in packs_found_from:
+            packs_found_from.remove(COMMON_TYPES_PACK)
+            common_types_pack_dependency = True
+        pack_dependencies_data = PackDependencies._label_as_optional(packs_found_from)
+        if common_types_pack_dependency:
+            pack_dependencies_data.extend(PackDependencies._label_as_mandatory({COMMON_TYPES_PACK}))
+        return pack_dependencies_data
```

Comment:
It is not clear what this function does. update the description and the param name/description `packs_found_from`
And why do we exclude CommonTypes?

Reply:
Say that I release a pack, Zimperium, which fetches incidents and has a classifier and a mapper.
the mapper maps to an incident field called `deviceid` which i've added to the `CommonTypes` pack.
only a mandatory dependency will coerce the customers to update their `CommonTypes` pack and will allow a smooth fetch process.

I'll update the desc and var names.

Reply:
OK, so you want the customer constantly to update their CommonTypes pack. Got it